src/overlays/actors/ovl_En_Dekubaba/z_en_dekubaba.c

```diff
@@ -121,85 +133,1191 @@ static ColliderJntSphInit D_809E8FC8 = {
+        // (Of course they reckoned without each age being able to use the other's items, so Stick and Master Sword
+        // jumpslash can give the Stick drop as adult, and neither will as child.)
+        if (LINK_IS_CHILD) {
+            sBigDekuBabaDamageTable.table[0x1B] = DMG_ENTRY(4, DEKUBABA_DMGEFF_NONE); // DMG_JUMP_MASTER
+        }
+
+        CollisionCheck_SetInfo(&this->actor.colChkInfo, &sBigDekuBabaDamageTable, &sColChkInfoInit);
+        this->actor.colChkInfo.health = 4;
+        this->actor.naviEnemyId = 8; // Big Deku Baba
+        this->actor.targetMode = 2;
+    } else {
+        this->size = 1.0f;
+
+        for (i = 0; i < sJntSphInit.count; i++) {
+            this->collider.elements[i].dim.worldSphere.radius = this->collider.elements[i].dim.modelSphere.radius;
+        }
+
+        if (LINK_IS_CHILD) {
+            sDekuBabaDamageTable.table[0x1B] = DMG_ENTRY(4, DEKUBABA_DMGEFF_NONE); // DMG_JUMP_MASTER
+        }
+
+        CollisionCheck_SetInfo(&this->actor.colChkInfo, &sDekuBabaDamageTable, &sColChkInfoInit);
+        this->actor.naviEnemyId = 7; // Deku Baba
+        this->actor.targetMode = 1;
+    }
+
+    EnDekubaba_SetupWait(this);
+    this->timer = 0;
+    this->boundFloor = NULL;
+    this->bodyPartsPos[3] = this->actor.home.pos;
+}
+
+void EnDekubaba_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnDekubaba* this = THIS;
+
+    Collider_DestroyJntSph(globalCtx, &this->collider);
+}
+
+void EnDekubaba_DisableHitboxes(EnDekubaba* this) {
+    s32 i;
+
+    for (i = 1; i < 7; i++) {
+        this->collider.elements[i].info.bumperFlags &= ~BUMP_ON;
+    }
+}
+
+void EnDekubaba_SetupWait(EnDekubaba* this) {
+    s32 i;
+    ColliderJntSphElement* element;
+
+    this->actor.shape.rot.x = -0x4000;
+    this->stemSectionAngle[0] = this->stemSectionAngle[1] = this->stemSectionAngle[2] = this->actor.shape.rot.x;
+
+    this->actor.world.pos.x = this->actor.home.pos.x;
+    this->actor.world.pos.z = this->actor.home.pos.z;
+    this->actor.world.pos.y = this->actor.home.pos.y + 14.0f * this->size;
+
+    Actor_SetScale(&this->actor, this->size * 0.01f * 0.5f);
+
+    this->collider.base.colType = COLTYPE_HARD;
+    this->collider.base.acFlags |= AC_HARD;
+    this->timer = 45;
+
+    for (i = 1; i < 7; i++) {
+        element = &this->collider.elements[i];
+        element->dim.worldSphere.center.x = this->actor.world.pos.x;
+        element->dim.worldSphere.center.y = (s16)this->actor.world.pos.y - 7;
+        element->dim.worldSphere.center.z = this->actor.world.pos.z;
+    }
+
+    this->actionFunc = EnDekubaba_Wait;
+}
+
+void EnDekubaba_SetupGrow(EnDekubaba* this) {
+    s32 i;
+
+    Animation_Change(&this->skelAnime, &gDekuBabaFastChompAnim,
+                     Animation_GetLastFrame(&gDekuBabaFastChompAnim) * (1.0f / 15), 0.0f,
+                     Animation_GetLastFrame(&gDekuBabaFastChompAnim), 2, 0.0f);
```

Comment:
```suggestion
                     Animation_GetLastFrame(&gDekuBabaFastChompAnim), ANIMMODE_ONCE, 0.0f);
```
also for other `Animation_Change` calls